src/test/java/guitests/BoardPickerTest.java

```diff
@@ -0,0 +1,98 @@
+package guitests;
+
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.VBox;
+import org.junit.Test;
+import org.loadui.testfx.GuiTest;
+import prefs.Preferences;
+import ui.IdGenerator;
+import ui.TestController;
+import ui.UI;
+import ui.listpanel.ListPanel;
+import util.PlatformEx;
+
+
+import static org.junit.Assert.assertEquals;
+import static ui.components.KeyboardShortcuts.SHOW_BOARD_PICKER;
+
+public class BoardPickerTest extends UITest {
+
+    private static final String boardNameInputId = IdGenerator.getBoardNameInputFieldIdReference();
+    private static final String boardNameSaveButtonId = IdGenerator.getBoardNameSaveButtonIdReference();
+
+    @Test
+    public void testBoardPicker() {
+        Preferences prefs = TestController.getUI().prefs;
```

Comment:
It should be better to have a comment explaining what's being tested, or if there are multiple action sequences below, then it's helpful to have a brief comment on the intention of each sequence also, so that it's easier for others to fix the test in the future.